google-cloud-storage/src/main/java/com/google/cloud/storage/WriteFlushStrategy.java

```diff
@@ -121,6 +146,27 @@ private static WriteObjectRequest possiblyPairDownRequest(
     return b.build();
   }
 
+  private static BidiWriteObjectRequest possiblyPairDownBidiRequest(
```

Comment:
Please add unit tests for this helper.

Reply:
My original intention was to introduce `@VisibleForTesting` for the specific comment thread around removing checksum in intermediate messages which is a misunderstanding on my part after talking with Jesse.

![image](https://github.com/googleapis/java-storage/assets/1907648/7b97e864-14f7-4a92-a16a-20b6ad50e328)

States that checksum can only be provided in first or last message sent to API so this comment and the one below are obsolete.